services/admin/tests/test_app_real.py

```diff
@@ -45,7 +45,7 @@ def test_force_refresh(
 ) -> None:
     dataset = "glue"
     processing_graph = ProcessingGraph(real_app_config.processing_graph.specification)
-    first_step = processing_graph.get_processing_steps()[0]
+    first_step = processing_graph.get_processing_steps(order="topological")[0]
```

Comment:
i added `"topological"` here because otherwise the first step was not `dataset-config-names` but first alphabetically

Reply:
Well seen